src/main/java/com/aws/greengrass/clientdevices/auth/certificate/infra/BackgroundCertificateRefresh.java

```diff
@@ -93,6 +95,7 @@ public void start(int intervalSeconds) {
            return;
         }
 
+        networkState.registerHandler(this);
```

Comment:
Register in infra init?

Reply:
It seemed kind of weird putting it there given already have access to the service here, but I can so that we keep registration in one place